content/source/docs/enterprise/sentinel/import/tfconfig.html.md

```diff
@@ -588,14 +552,43 @@ main = rule { subject.outputs.id.sensitive }
 * **Value Type:** Any primitive type, list or map.
 
 The `value` value within the [output namespace](#namespace-outputs) represents
-the defined value for the output as declared in the configuration.
+the defined value for the output as declared in the configuration.  Primitives
+will bear the implicit type of its declaration (string, int, float, or bool),
+and maps and lists will be represented as such.
 
-The actual value will be as configured. Primitives will bear the implicit type
-of its declaration (string, int, float, or bool), and maps and lists will be
-represented as such.
+**With Terraform 0.11 or earlier**, if the output value is defined as an
+expression (and not a static value), the value will be in its raw,
+non-interpolated string. **With Terraform 0.12 or later**, any non-static
+values (such as interpolated strings) are not present and
+[`references`](#value-references) should be used instead.
+
+As an example, given the following output declaration block:
+
+```hcl
+output "id" {
+  value = "${null_resource.foo.id}"
+}
+```
+
+With Terraform 0.11 or earlier the following policy would evaluate to `true`:
 
-More often than not, the value will show up in its raw, non-interpolated string,
-unless the output is defined to a static value.
+```python
+import "tfconfig"
+
+main = rule { tfconfig.outputs.id.value is "${null_resource.foo.id}" }
+```
+
+For the same sort of policy with Terraform 0.12 or later, see the
+[references](#value-references) documentation below.
+
+### Value: `references`
+
+* **Value Type:**. List of strings.
+
+-> Note: This value is only present when using Terraform 0.12 or later
+
+The `references` value within the [output namespace](#namespace-outputs)
+represents the identifier within an expression found in [`value`](#value-value).
```

Comment:
I think this needs a little more elaboration; what's here doesn't quite tell me what I need to know to use it. 

My GUESS, from extrapolating from what I'm seeing here:

- Either:
    - If the value is any expression other than a literal value, `value` is always empty. This includes concatenating two literals together with a function, or something. 
    - If the value is an expression that _references one or more named values,_ `value` is always empty. Other kinds of transformations, like functions, would still end up in the `value`.
- And either:
    - `references` is a list of every named value that was referenced in the value's expression. This is extracted using the same sort of logic Terraform uses to infer dependencies. 
    - `references` is a list of every... actually, IDK what it would be otherwise, but if `value` works like the first option above, I'd expect something odd.

Reply:
Hey Nick, Chris and I talked about this offline today and it it is being updated to elaborate and also update the other fields where references exists. This should help address this.

Reply:
OK, great, thanks!